### PR TITLE
Add new character information from v5.2 Phase 1 and their test cases

### PR DESCRIPTION
**Pull request by sdglitched**
Add new character information from v5.2 Phase 1

- Chasca
Screenshot From 2024-11-23 18-38-12
Screenshot From 2024-11-23 18-38-36

- Ororon
Screenshot From 2024-11-23 19-29-03
Screenshot From 2024-11-23 19-29-15

**Note**: The character information like ATK, DEF and HP are yet to be updated as these values are still not available Genshin Impact Wiki. Also, test cases will added once all the information is available.

Fixes: #221

### CODE REVIEW

**Comment by sdglitched** [comment]
Data is now updated in Genshin Wiki. :rocket: 

- Chasca
Screenshot From 2024-11-24 20-58-07
Screenshot From 2024-11-24 20-47-35

- Ororon
Screenshot From 2024-11-24 20-57-56
Screenshot From 2024-11-24 20-55-30

Test Coverage
Screenshot From 2024-11-24 21-12-11

File: assets/cons.qrc
Comment (gridhead) [inline assets/cons.qrc:1]: Could you please add the constellation details for the Chasca?

**Review by gridhead — CHANGES_REQUESTED** [review]
Apart from Chasca's constellation, this looks good to me.

**Review by gridhead — APPROVED** [review]
I am going ahead with the merge but this needs following up.

Related issue ticket. #227

**Pull request merged**